Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Making ldlinux quiet by enabling some output only in verbose mode.

diff -Naurp syslinux.orig/core/diskstart.inc syslinux/core/diskstart.inc
--- syslinux.orig/core/diskstart.inc
+++ syslinux/core/diskstart.inc
@@ -111,8 +111,10 @@ ldlinux_ent:
 ;
 ; Tell the user we got this far
 ;
+%ifdef VERBOSE_MESSAGES
 		mov si,early_banner
 		call writestr_early
+%endif
 
 ;
 ; Checksum data thus far
@@ -138,8 +140,10 @@ print_bios:
 		mov si,ebios_name
 		mov byte [getlinsec.jmp+1],(getlinsec_ebios-(getlinsec.jmp+2))
 .cbios:
+%ifdef VERBOSE_MESSAGES
 		mov [BIOSName],si
 		call writestr_early
+%endif
 
 		section .earlybss
 		global BIOSName
@@ -466,11 +470,13 @@ all_read:
 ; Let the user (and programmer!) know we got this far.  This used to be
 ; in Sector 1, but makes a lot more sense here.
 ;
+%ifdef VERBOSE_MESSAGES
 		mov si,late_banner
 		call writestr_early
 
 		mov si,copyright_str
 		call writestr_early
+%endif
 
 
 ;
